assets/js/app.js

```diff
@@ -382,9 +392,14 @@ window.addEventListener("storage", function () {
 
 // Function to remove scroll bar during preload
 $(window).on('load', function() {
-
-  $('.loader-container').fadeOut(2500, function() {
-    $('.no-scroll-preload').css('overflow', 'visible');
```

Comment:
Please pull the recent changes from the master branch and incorporate them into your PR